docs/cloudslang_dsl_reference.md

```diff
@@ -520,6 +520,32 @@ publish:
   - total_cost: fromInputs['total_cost'] + cost
 ```
 
+##get
+May appear in the value of an [input](#inputs), [output](#outputs), [publish](#publish) or [loop expression](#for).
+
+The function in the form of `get('key', 'default_value')` returns the value associated with `key` if the key is defined and its value is not `None`. If the key is undefined or its value is `None` the function returns the `default_value`.
+
+**Example - usage of get function in inputs and outputs**
+
+```yaml
+inputs:
+  - input1:
+      required: false
+  - input1_safe:
+      default: get('input1', 'default_input')
+      overridable: false
+      required: false
```

Comment:
I would remove `required: false` from here -- so the user will not be confused (the value of `input1_safe` cannot be `None`)

I used `required: false` in my tests so the test case would fail with a meaningful error message in the `Assert` (valiadation) section if the `get` function is not working properly (e.g. returns None) -- if I would have left the input `required: true` the flow would fail in input binding stage

WDYT?